bin/fk.sh

```diff
@@ -35,7 +35,11 @@ fpr()
       # assuming ssh remote since `fclone` clone from ssh remote by default
       repoName=$(git config --get remote.origin.url | xargs basename | cut -d'.' -f1)
       ownerName=$(git config --get remote.origin.url | cut -d'/' -f1 | cut -d':' -f2)
-      xdg-open https://github.com/$ownerName/$repoName/pull/new/$curBranch &> /dev/null
+      if pr-exists.rb repoName ownerName curBranch; then
+        xdg-open https://github.com/$ownerName/$repoName/pull/new/$curBranch &> /dev/null
```

Comment:
does linux support `open` because then that will compatible with both and mac and linux

Reply:
Nope :cry:  i actually looked it up, `xdg-open` is Linux's equivalent of macs's `open`. I will make a separate issue for mac support

Reply:
We can have something like 



```bash 
ifLinuxRun()
{
	if [ "Linux" -eq $(uname) ]; then
		eval $2
	fi
}

ifMacRun()
{
	if [ "Darwin" -eq $(uname) ]; then
		eval $2
	fi
}

osSpecificCommand()
{
    case $1 in
		open)
			shift;
			ifLinuxRun "xdg-open $1"
			ifMacRun "open $1"
		*)
			echo "Invalid Command"
			exit 1
}

osSpecificCommand open git....
```